services/web/server/tests/unit/with_dbs/04/functions_rpc/test_functions_controller_rpc.py

```diff
@@ -496,48 +727,21 @@ async def test_list_function_jobs_for_functionid(
     assert all(j.function_uid == first_registered_function.uid for j in jobs)
 
 
-async def test_delete_function_job(
```

Comment:
Q: does deleting not applu any longer?

Reply:
Sorry, I don't think i understand your question.

Reply:
I'm gonna close this not to block the PR. But feel free to chat to me about it.